File: api/code-samples/javascript/klaytnDebug/vMTracing/traceBlockByNumber.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const block = 1449;

    sdk.debug.traceBlockByNumber(block, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/vMTracing/traceBlockByNumber.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

number = 21

sdk = OpenSDK(host)
debug_response = sdk.debug.trace_block_by_number(number)

print(json.loads(debug_response.response.data))
